src/components/NewMessageModal.vue

```diff
@@ -294,6 +294,51 @@ export default {
 		},
 		toHtml,
 		plain,
+		async onNewSentMailbox(data, account) {
+			showWarning(t('mail', 'Setting Sent default folder'))
+			let newSentMailboxId = null
+			const mailboxes = this.mainStore.getMailboxes(data.accountId)
+			const sentMailboxId = mailboxes.find((mailbox) => mailbox.name === account.personalNamespace + 'Sent' || mailbox.name === account.personalNamespace + t('mail', 'Sent'))?.databaseId
+			if (sentMailboxId) {
+				await this.setSentMailboxAndResend(account, sentMailboxId, data)
+			}
+			logger.info(`creating ${t('mail', 'Sent')} mailbox`)
+			try {
+				const newSentMailbox = await this.mainStore.createMailbox({ account, name: account.personalNamespace + t('mail', 'Sent') })
```

Comment:
if the IMAP server supports it (likely) we should also set the specialuse attribute to `\sent`